app/services/opd_service/report_engine.rb

```diff
@@ -182,6 +182,20 @@ def registration(start_date, end_date)
       return stats
     end
 
+    def attendance(start_date, end_date)
```

Comment:
Add this under REPORTS in OPDService::ReportEngine then add a class or module that handles this